softwareconfiguration/mlplan/full/testrsc/report/statistics-report/regression.json

```diff
@@ -0,0 +1,8 @@
+{
+  "num_evaluations" : 0,

```

Comment:
Are all of these statistics file supposed to be in here?

Reply:
yes, they are used to assert that the statistics reports are correctly produced from the results obtained by ml-plan.